docs/04-architecture/turbopack.mdx

```diff
@@ -22,7 +22,145 @@ Turbopack can be used in Next.js in both the `pages` and `app` directories for f
 
 ## Supported Features
 
-To learn more about the currently supported features for Turbopack, view the [documentation](https://turbo.build/pack/docs/features).
+Turbopack can be customized to transform different files and change how modules are resolved. It supports a subset of webpack's loader API and offers similar configuration aliasing module resolution.
+
+> **Good to know**:
+>
+> Turbopack for Next.js does not require loaders nor loader configuration for
+> built-in functionality, just as they aren't required for Next.js. Turbopack
+> has built-in support for css and compiling modern JavaScript, so there's no
+> need for `css-loader`, `postcss-loader`, or `babel-loader` if you're just
+> using `@babel/preset-env`.
+
+If you need loader support beyond what's built in, many webpack loaders already work with Turbopack. There are currently some limitations:
+
+- Only a core subset of the webpack loader API is implemented. This is enough for some popular loaders, and we'll expand our support for this API in the future.
+- Only loaders that return JavaScript code are supported. Loaders that transform files like stylesheets or images are not currently supported.
+- Options passed to webpack loaders must be plain JavaScript primitives, objects, and arrays. For example, it's not possible to pass `require()`d plugin modules as option values.
+
+As of Next 13.2, configuring webpack loaders is possible for Next.js apps through an experimental option in `next.config.js`. `turbo.rules` can be set as a mapping of file globs to a list of package names or `{loader, options}` pairs:
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    turbo: {
+      rules: {
+        // Option format
+        '*.md': [
+          {
+            loader: '@mdx-js/loader',
+            options: {
+              format: 'md',
+            },
+          },
+        ],
+        // Option-less format
+        '*.mdx': ['@mdx-js/loader'],
+      },
+    },
+  },
+}
+```
+
+> **Good to know**:
+>
+> Note: Prior to Next.js version 13.4.4, `experimental.turbo.rules` was named
+> `experimental.turbo.loaders` and only accepted file extensions like `.mdx`
+> instead of `*.mdx`.
```

Comment:
```suggestion

> **Good to know**: Prior to Next.js version 13.4.4, `experimental.turbo.rules` was named `experimental.turbo.loaders` and only accepted file extensions like `.mdx` instead of `*.mdx`.
```